docs: anonymous sign-ins (#21801)
supabase/supabase: apps/docs/content/guides/auth/row-level-security.mdx

using ( true );
```

> <Admonition>

Reviewer: Add a label / note thing?

> The postgres `anon` role is different from an [anonymous user](/docs/guides/auth/auth-anonymous) in Supabase Auth. An anonymous user uses the `authenticated` role and can be differentiated from a permanent user by checking the `is_anonymous` claim in the JWT using `auth.jwt()` in the RLS policies.

Reviewer: ```suggestion
Using the `anon` Postgres role is different from an [anonymous user](/docs/guides/auth/auth-anonymous) in Supabase Auth. Those users access the database via the `authenticated` role and can be differentiated from a permanent user by checking the `is_anonymous` claim in the JWT.
```
